dev_guide/builds/build_inputs.adoc

```diff
@@ -731,86 +731,115 @@ archive, and, if valid, the builder changes into that subdirectory before
 executing the build.
 
 [[using-secrets-during-build]]
-== Input Secrets
+== Input Secrets and Config Maps
 
-In some scenarios, build operations require credentials to access dependent
-resources, but it is undesirable for those credentials to be available in the
-final application image produced by the build. You can define _input secrets_
-for this purpose.
+In some scenarios, build operations require credentials or other configuration 
+data to access dependent resources, but it is undesirable for that information 
+to be placed in source control. You can define _input secrets_ and _input 
+configMaps_ for this purpose.
 
-For example, when building a Node.js application, you can set up your private
-mirror for Node.js modules. In order to download modules from that private
-mirror, you have to supply a custom *_.npmrc_* file for the build that contains
-a URL, user name, and password. For security reasons, you do not want to expose
-your credentials in the application image.
+For example, when building an Java application via Maven, you can set up a 
+private mirror of Maven Central or JCenter that is accessed via private keys.
+In order to download libraries from that private mirror, you have to supply the
+following:
 
-This example describes Node.js, but you can use the same approach for adding SSL
-certificates into the *_/etc/ssl/certs_* directory, API keys or tokens, license
-files, and more.
+. A  *_settings.xml_* file configured with the mirror's URL and connection 
+settings.
+. A private key referenced in the settings file, such as *_~/.ssh/id_rsa_*.
+
+For security reasons, you do not want to expose your credentials in the 
+application image.
+
+This example describes a Java application, but you can use the same approach 
+for adding SSL certificates into the *_/etc/ssl/certs_* directory, API keys or 
+tokens, license files, and more.
 
 [[using-secrets-in-the-buildconfig]]
-=== Adding Input Secrets
+=== Adding Input Secrets and ConfigMaps
 
-To add an input secret to an existing `BuildConfig`:
+To add an input secret and/or config map to an existing `BuildConfig`:
+
+. Create the config map, if it does not exist:
```

Comment:
s/config map/configmap ?

Reply:
I'll defer to @bmcelvee on conventions around kube objects like `ConfigMap`

Reply:
Let's go with ConfigMap.